migrations/900000000000004_user_management.up.sql

```diff
@@ -0,0 +1,156 @@
+BEGIN;
```

Comment:
let's prefix all the functions/views added here with a common prefix like `user_management` for consistency/easier finding